lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -175,6 +175,16 @@ const FeatureInfoSection = createReactClass({
 
     render() {
         const catalogItemName = (this.props.catalogItem && this.props.catalogItem.name) || '';
+        let baseFilename = catalogItemName;
+        if (this.props.position) {
+            const position = Ellipsoid.WGS84.cartesianToCartographic(this.props.position);
+            const latitude = CesiumMath.toDegrees(position.latitude).toFixed(5);
+            const longitude = CesiumMath.toDegrees(position.longitude).toFixed(5);
+            // Check that the catalogItemName doesn't already contain the lat lon with the same or more precision.
+            if ((typeof baseFilename !== 'string') || (baseFilename.indexOf(latitude) === -1) || (baseFilename.indexOf(longitude) === -1)) {
```

Comment:
This test may not be completely reliable, because `toFixed` will round the last digit.  e.g. `(11.111116).toFixed(5) === '11.11112'`.